### ISSUE TITLE
BUG:Installation errors on aarch64

### ISSUE THREAD

**Issue by tathastu871** [pandas-dev/pandas#35932]
- [+] I have checked that this issue has not already been reported.

- [+] I have confirmed this bug exists on the latest version of pandas.

- [+] (optional) I have confirmed this bug exists on the master branch of pandas.

---

**Note**: Please read [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) detailing how to provide the necessary information for us to reproduce your bug.

#### Code Sample, a copy-pastable example

```python
# Your code here
pip3 install pandas
or
pip3 install pandas --no-build-isolation --no-binary :all:
or
python setup.py build_ext --inplace --force
or 
LDFLAGS=" -lm -lcompiler_rt" pip3 install pandas --no-build-isolation --no-binary :all: --no-use-pep517
```

#### Problem description
```https://termbin.com/wzsr```
[this should explain **why** the current behaviour is a problem and why the expected output is a better solution]

#### Output of ``pd.show_versions()``
When installed from source instead of pip it installs pandas but no submodules
Output of pd.show_versions()
```AttributeError: module 'pandas' has no attribute 'show_versions'```

<details>

[paste the output of ``pd.show_versions()`` here leaving a blank line after the details tag]

</details>

**Comment by bashtage** [comment]
Can you post the relevant issue from the build log?

**Comment by tathastu871** [comment]
> Can you post the relevant issue from the build log?
> 
> What is `https://termbin.com/wzsr`?

i have pasted  build log on that site

**Comment by tathastu871** [comment]
Initially errors related to cython arised even when its installed.
After changing commands it solved but somekind of deprecation eŕror arised as shown in build log.

However when installed directly from source pandas seems to installed even with errors and imported properly but no sub modules were installed. 

Means if i use pandas.read_html or excel or other it says no such attributes.

I was able to use previous versions of pandas but on latest version such problem arised

**Comment by bashtage** [comment]
Why are you using `--no-binary`?  This appears to be blocking it from building a wheel of pandas.

**Comment by bashtage** [comment]
What is the log you see from

`
python setup.py build_ext --inplace --force
`

**Comment by tathastu871** [comment]
> Why are you using `--no-binary`? This appears to be blocking it from building a wheel of pandas.
As other options failed. gave it a try. But no success

**Comment by tathastu871** [comment]
> What is the log you see from
> 
> `python setup.py build_ext --inplace --force`

```http://sprunge.us/5v0EuG```
Pasted on above

**Comment by tathastu871** [comment]
> What is the log you see from
> 
> `python setup.py build_ext --inplace --force`

**Comment by tathastu871** [comment]
Temporary solution:
Pandas 1.0.5 seems to be working. 
However workaround and patching needed for latest 1.1.1 version. 

Please implement necessary changes to make latest pandas work on aarch64

**Comment by xeffyr** [comment]
@tathastu871 Try this:
```
export CFLAGS="-Wno-deprecated-declarations -Wno-unreachable-code"
pip install pandas
```

**Comment by tathastu871** [comment]
> @tathastu871 Try this:
> 
> ```
> export CFLAGS="-Wno-deprecated-declarations -Wno-unreachable-code"
> pip install pandas
> ```
> 
> - fixes [#35932 (comment)](https://github.com/pandas-dev/pandas/issues/35932#issuecomment-682082619)

Thanks man it worked.
